marklogic-data-hub-central/ui/e2e/cypress/support/components/model/graph-view-side-panel.tsx

```diff
@@ -42,6 +42,19 @@ class GraphViewSidePanel {
   getPersonEntityPrefix() {
     return cy.findByTestId("prefix");
   }
+
+  selectGraphViewTab() {
```

Comment:
This seems like something which should be defined in the parent component Model.tsx within support/pages. Also, looks like there is already a function selectView() within Model.tsx which is being used in few places to select a particular view in Model tile. Let's just use one of these two functions and remove the other one.

Reply:
Thanks for the comment; replaced it with selectView().